Resources/public/js/views/services/ez-trashviewservice.js

```diff
@@ -223,6 +223,54 @@ YUI.add('ez-trashviewservice', function (Y) {
         },
 
         /**
+         * Restores trash items
+         *
+         * @method _restoreItems
+         * @protected
+         * @param {Object} e restoreItems event facade
+         * @param {Array} e.trashItems  List of trashItems to be restored
+         */
+        _restoreItems: function (e) {
+            var loadOptions = {api: this.get('capi')},
+                tasks = new Y.Parallel(),
+                service = this,
+                notificationNamePrefix = "restoreTrashItem-";
+
```

Comment:
I pinged PM about it in the Jira issue.
We still need to have a single notification if only one item is selected as we will later display the name and a link to the content. We could have a dedicated notification behavior if multiple items are selected.